packages/client/src/arpa_reporter/views/Home.vue

```diff
@@ -105,7 +113,42 @@ export default {
       this.sending = true;
 
       try {
-        const result = await getJson('/api/audit_report?queue=true');
+        const periodId = this.$store.getters.viewPeriod.id || 0;
+        const periodIdQuery = (periodId && !this.viewingOpenPeriod) ? `&period_id=${periodId}` : '';
+        const url = `/api/audit_report?queue=true${periodIdQuery}`;
+        const result = await getJson(url);
+
+        if (result.error) {
+          this.alert = {
+            text: 'Something went wrong. Unable to send an email containing the audit report. Reach out to grants-helpdesk@usdigitalresponse.org if this happens again.',
+            level: 'err',
+          };
+          console.log(result.error);
+        } else {
+          this.alert = {
+            text: 'Sent. Please note, it could take up to 1 hour for this email to arrive.',
+            level: 'ok',
+          };
+        }
+      } catch (error) {
+        // we got an error from the backend, but the backend didn't send reasons
+        this.alert = {
+          text: 'Something went wrong. Unable to send an email containing the audit report. Reach out to grants-helpdesk@usdigitalresponse.org if this happens again.',
+          level: 'err',
+        };
+      }
+
+      this.sending = false;
+    },
+    async downloadAuditReport() {
+      this.sending = true;
+
+      try {
+        const periodId = this.$store.getters.viewPeriod.id || 0;
+        const periodIdQuery = (periodId && !this.viewingOpenPeriod) ? `&period_id=${periodId}` : '';
+        const url = `/api/audit_report?async=false${periodIdQuery}`;
+        const result = await getJson(url);
+        // const result = await getJson('/api/audit_report?async=false');
 
         if (result.error) {
           this.alert = {
```

Comment:
The error messages in this function still refer to "sending an email".  Is that correct?

What's the difference between "send audit report" and "download audit report" now?

Reply:
Good catch. This is most likely a merge issue from changes made since I first made this PR. I think this PR is originally from before we switched to the separate worker.

There is some handling in the code for a download audit report button that is used for testing. The difference is that it runs on the server, not the separate sqs worker. I was using that for testing before I setup all the docker / localstack pieces.

I'll fix this part up.

Reply:
Looking at this again, this is correct. I just added an alert that we didn't have before.

In the current workflow, the generate report button creates a message on SQS. That message is picked up by a worker which runs the report, puts it on s3, then sends an email with the link to download the report from s3.
You can see the code looking at `processSQSMessageRequests` in `lib/audit-report.js`, which then calls `generateAndSendEmail`.

That being said, I don't think this alert / error is part of this issue, so I could safely remove it.

Thoughts?